frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -315,6 +328,147 @@ declare global {
         response: OdhResponse<RegisteredModelList | undefined>,
       ): Cypress.Chainable<null>;
 
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId/versions/:pipelineVersionId`,
+        options: { path: { namespace: string; pipelineId: string; pipelineVersionId: string } },
+        response: OdhResponse<PipelineVersionKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId/versions/:pipelineVersionId`,
+        options: {
+          path: { namespace: string; pipelineId: string; pipelineVersionId: string };
+          times?: number;
+        },
+        response: OdhResponse<GenericStaticResponse<string, Body> | K8sStatus>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId`,
+        options: { path: { namespace: string; pipelineId: string } },
+        response: OdhResponse<PipelineKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/upload_version`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineVersionKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId`,
+        options: { path: { namespace: string; pipelineId: string } },
+        response: OdhResponse<PipelineKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId/versions`,
+        options: { path: { namespace: string; pipelineId: string }; times?: number },
+        response: OdhResponse<PipelineVersionKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId/versions`,
+        options: { path: { namespace: string; pipelineId: string } },
+        response: OdhResponse<ListPipelineVersionsKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelinesResponseKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns/:recurringRunsId`,
+        options: { path: { namespace: string; recurringRunsId: string } },
```

Comment:
Update all occurrences.
```suggestion
        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns/:recurringRunId`,
        options: { path: { namespace: string; recurringRunId: string } },
```